app/__init__.py

```diff
@@ -1,7 +1,13 @@
 from flask import Flask
 from .db import db, migrate
 from .models import task, goal
+from .routes.task_routes import tasks_bp
 import os
+from dotenv import load_dotenv
+from .routes.goal_routes import goals_bp
+
+load_dotenv()
+SLACKBOT_TOKEN = os.environ.get("SLACKBOT_TOKEN")
```

Comment:
Because we aren't using the Slackbot token in this file, we don't need to bring it in here!